schemas/vertices/gtdb/gtdb_taxon.json

```diff
@@ -1,23 +1,92 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$schema": "http://json-schema.org/draft-xxx/schema#",
   "type": "object",
-  "description": "A entry in the GTDB taxonomy tree.",
-  "required": ["_key", "type", "name"],
+  "description": "Template for a vertex entry in the NCBI or GTDB taxonomy tree.",
+  "required": ["_key", "source", "scientific_name", "rank"],
   "properties": {
     "_key": {
       "type": "string",
       "description": "Taxon type abbreviation plus name",
       "examples": ["d:Bacteria", "p:Firmicutes"]
     },
-    "name": {
+    "source": {
+      "type": "string",
+      "title": "Taxonomy source",
+      "examples": ["NCBI", "GTDB"]
+    },
+    "scientific_name": {
       "type": "string",
       "title": "Taxon name.",
-      "examples": ["Bacteria", "Firmicutes"]
+      "examples": ["Methylophilus methylotrophus", "Bacteria", "Firmicutes"]
+    },
+    "canonical_scientific_name": {
+      "type": "array",
+      "title": "Canonicalized scientific name",
+      "examples": [ [ "methylophilus", "methylotrophus" ], 
+                    ["Bacteria"], 
+                    ["Firmicutes"]
+                  ],
+      "items": { "type": "string" }
+    },
+    
+    "aliases": {
+      "type": "array",
+      "description": "Aliases",
+      "examples": [ 
+                    [ {"category": "authority", 
+                       "name": "Borreliella burgdorferi (Johnson et al. 1984) Adeolu and Gupta 2015",
+                       "canonical": ["borreliella", "burgdorferi", "johnson", "adeolu", "gupta" ]},
+                      {"category": "genbank common name", 
+                       "name":"Lyme disease spirochet",
+                       "canonical": ["lyme", "disease", "spirochet"] },
+                      {"category":"synonym", 
+                       "name":"Borrelia burgdorferi",
+                       "canonical": [ "borrelia", "burgdorferi" ]} 
+                       ],
+
+                    [ {"category": "common name",
+                       "name": "E. coli",
+                       "canonical": ["e", "coli"] },
+                      {"category": "authority", 
+                       "name": "\"Bacterium coli commune\" Escherich 1885",
+                       "canonical": ["bacterium", "coli", "commune", "escherich"] },
+                      {"category": "synonym",
+                       "name": "Bacterium coli",
+                       "canonical": ["bacterium", "coli" ] } ]
+                    ]
+                  ],
```

Comment:
Stray comma here, json doesn't let you do trailing commas

Reply:
whoops